cfme/tests/automate/test_method.py

```diff
@@ -492,11 +496,11 @@ def test_embedded_method_selection():
         expectedResults:
             1.
             2. Selected embedded method should be visible
-
-    Bugzilla:
-        1718495
     """
-    pass
+    path = ("Datastore", "ManageIQ (Locked)", "System", "CommonMethods", "Utils", "log_object")
+    view = navigate_to(klass.methods, "Add")
+    view.fill({'location': "Inline", "embedded_method": path})
+    assert view.embedded_method_table.read()[0]['Path'] == "/" + "/".join(path[2:])
```

Comment:
Please don't use `+` for string concatenation, use an f-string or a `format` call.